.github/workflows/tests.yml

```diff
@@ -202,6 +202,15 @@ jobs:
   test-integration-gcp:
     name: Integration - Linux, GCP
     runs-on: ubuntu-20.04
+    container:
```

Comment:
I'm so happy you did this! Yes!

Reply:
I cannot work out why service containers are not working for some of the jobs though. It's very weird. They are failing on network calls to localhost. And GH's docs are woeful. I am going to have to ask some GH folks I think unless inspiration strikes today.

Reply:
:cry: Want to pair a bit on this?

Reply:
I've found the issue. :( The service container syntax is a half-bake implementation of something Docker Compose lite/like/ish. Passing options to it doesn't work like you'd expect and so it's not doing any ENTRYPOINT overriding which means most of the services aren't starting right.